WORKSPACE

```diff
@@ -54,8 +57,13 @@ skydoc_repositories()
 load("@io_bazel_rules_sass//:package.bzl", "rules_sass_dependencies")
 rules_sass_dependencies()
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "npm_install")
 node_repositories()
+npm_install(
```

Comment:
Are these are example-only deps? Might be helpful to have a comment above them. I know we haven't done this historically, but I think it will become increasingly more helpful to have documentation as the surface area of rules_rust grows.

Reply:
Yep that's correct. Added a comment.